packages/core/config.d.ts

```diff
@@ -62,4 +62,11 @@ export interface Config {
       timezone: string;
     }[];
   };
+  auth?: {
+    /**
+     * The environment config added to be able to change the authentication environment.
```

Comment:
Bit of a change of wording here too, I think. Like "What environment parameter to use when communicating with the auth backend." and same as above, mentioning the typical values and what the default value is.

Reply:
I have rephrased it! let me know if it needs further changes